static/js/util/api.js

```diff
@@ -9,7 +9,7 @@ function getCookie(name) {
   if (document.cookie && document.cookie !== '') {
     let cookies = document.cookie.split(';');
 
-    for (var i = 0; i < cookies.length; i++) {
+    for (let i = 0; i < cookies.length; i++) {
```

Comment:
Can we change this to `for (let cookie of cookies) {`? This code is adapted from the Django docs which are unaware of ES6